src/components/Concentrator/ConcentratorVaultTable.tsx

```diff
@@ -81,3 +81,15 @@ export const ConcentratorVaultTable: FC<ConcentratorVaultTableProps> = ({
     </VaultTable>
   )
 }
+
+const ConcentratorVaultRow: FC<ConcentratorVaultProps> = (props) => {
+  return (
+    <VaultRow
```

Comment:
You could just use this directly in the `map` on line 70 without declaring it as a component.